README.md

```diff
@@ -257,7 +249,39 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
 };
 ```
 
-<br />
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:
+
+```sh
+npx @unleash/nextjs
+```
+
+For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:
+
+- `UNLEASH_SERVER_API_URL`
+- `UNLEASH_SERVER_API_TOKEN`
+
+The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

Comment:
Realized this was missing a word 😅 

```suggestion
The CLI will attempt to read environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```

Reply:
I think I will drop shorthand ENV for simplicity.